pyproject.toml

```diff
@@ -118,20 +118,15 @@ replace = "[{new_version}] {now:%Y-%m-%d}"
 # ============================================================================
 
 [tool.black]
-# we should set this to 179
-line-length = 120
+line-length = 179
```

Comment:
There's a paragraph in the devguide that needs to be updated to reflect this change as well. And we need to create follow up issues to update this across the ecosystem (compas theme, and other core packages)

Reply:
In that line, I think the devguide needs to be update to indicate the move to `ruff`, right?

Reply:
> There's a paragraph in the devguide that needs to be updated to reflect this change as well. And we need to create follow up issues to update this across the ecosystem (compas theme, and other core packages)

am already doing this for `compas_cgal`, `compas_occ`, `compas_libigl`, `compas_gmsh`, `compas_viewer`, `compas_notebook`, `compas_model`

Reply:
re the devguide: yes indeed, we should. will make the change...